scripts/utils/fuels/relayCommonMessage.ts

```diff
@@ -0,0 +1,140 @@
+/// @dev The Fuel testing utils.
+/// A set of useful helper methods for the integration test environment.
+import {
+  Message,
+  WalletUnlocked as FuelWallet,
+  ZeroBytes32,
+  ScriptTransactionRequest,
+  TransactionRequestLike,
+  arrayify,
+  InputType,
+  hexlify,
+  OutputType,
+  TransactionResponse,
+  Predicate,
+} from 'fuels';
+import { readFileSync } from 'fs';
+import { join } from 'path';
+import { debug } from '../logs';
+
+const brigdePredicateBytecode = readFileSync(
+  join(__dirname, '../../../bridge-message-predicates/contract_message_predicate.bin')
+);
+const brigdeScriptBytecode = readFileSync(
+  join(__dirname, '../../../bridge-message-predicates/contract_message_script.bin')
+);
+
+// Create a predicate contract for common messages
```

Comment:
```suggestion
// Create a predicate for common messages
```

A predicate and a contract are two different program types